components/lock.rst

```diff
@@ -546,6 +608,46 @@ method uses the Memcached's ``flush()`` method which purges and removes everythi
     The method ``flush()`` must not be called, or locks should be stored in a
     dedicated Memcached service away from Cache.
 
+MongoDbStore
+~~~~~~~~~~~~
+
+.. caution::
+
+    The locked resource name is indexed in the ``_id`` field of the lock
+    collection. Beware that in MongoDB an indexed field's value can be
+    `a maximum of 1024 bytes in length`_ inclusive of structural overhead.
+
+A TTL index MUST BE used to automatically clean up expired locks.
```

Comment:
If bold then **must be**

Reply:
in the interest of consistency with all other usage of `must be` / `must not be` i've just made it lower case / not bold.